exonum/benches/criterion/lib.rs

```diff
@@ -16,20 +16,21 @@
 extern crate criterion;
 #[macro_use]
 extern crate exonum;
+extern crate exonum_crypto as crypto;
 extern crate futures;
 extern crate num;
 extern crate rand;
 extern crate tempdir;
 extern crate tokio_core;
 
+use bench_crypto::bench_crypto;
```

Comment:
I had to rename this mod because we need to use exonum_crypto as crypto.
I'm thinking if we can rename it back.